app/consumer-democracy/app.go

```diff
@@ -475,21 +532,37 @@ func New(
 
 	skipGenesisInvariants := cast.ToBool(appOpts.Get(crisis.FlagSkipGenesisInvariants))
 
+	keeperMap := map[string]interface{}{
+		"/cosmos.gov.v1.MsgUpdateParams":               app.GovKeeper,
+		"/cosmos.bank.v1beta1.MsgUpdateParams":         app.BankKeeper,
+		"/cosmos.staking.v1beta1.MsgUpdateParams":      &app.StakingKeeper,
+		"/cosmos.distribution.v1beta1.MsgUpdateParams": app.DistrKeeper,
+		"/cosmos.mint.v1beta1.MsgUpdateParams":         app.MintKeeper,
```

Comment:
Why not pull these directly from the sdk?

Reply:
It's outdated now, resolving